src/components/checkbox/checkbox.component.ts

```diff
@@ -68,6 +68,8 @@ export class CheckboxComponent<T = number> implements ControlValueAccessor {
     /** Used to inform Angular forms that the component value has changed */
     onChangeCallback: (_: boolean | T) => void = () => { };
 
+    private _changeDetector: ChangeDetectorRef;
```

Comment:
Does this not require a constructor to import the changeDetector like in other places below?